CONTRIBUTING.md

```diff
@@ -54,22 +54,35 @@ on GitHub if you don't have a token.
 
 All features should work, including building and launching.
 
-## Testing
+## Increase your GitHub API limit
 
-It is recommended to create and enable your GitHub API token before running tests
+By default, GitHub has a limit of 60 API requests per hour. Thus,
```

Comment:
Let's link to the GitHub page for clarity and more info: https://developer.github.com/v3/guides/getting-started/#authentication